pkg/config/database.go

```diff
@@ -11,14 +12,19 @@ import (
 	"github.com/jmoiron/sqlx/reflectx"
 	"github.com/pkg/errors"
 	"net"
+	"net/url"
+	"strconv"
 	"sync"
 	"time"
 )
 
+var ErrBadDBType = goerrors.New("unknown database type")
```

Comment:
I would make this a function that reports `unkown database type $type. Must be one of mysql or postgresql` and uses `errors.New()` (from "github.com/pkg/errors") in order to provide the correct stack.

Reply:
Let me guess: to follow already existing code patterns?

Reply:
> Let me guess: to follow already existing code patterns?

Yes.